summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -144,4 +144,4 @@ formatGhcMeta GhcMeta{..} =
     <> "  "
     <> T.justifyLeft 14 ' ' gmBase
     <> "  "
-    <> T.justifyLeft 18 ' ' gmResolver
```

Comment:
As this padding is not used for border creation it doesn't make much sense to keep it in the first place :)